DDDEastAnglia/Helpers/Security/IOAuthClientInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using DotNetOpenAuth.AspNet;
+using Microsoft.Web.WebPages.OAuth;
+
+namespace DDDEastAnglia.Helpers.Security
+{
+    public interface IOAuthClientInfo
```

Comment:
I don't agree - the interface was introduced in this pull request, so redefining it to be a number of smaller, more cohesive interfaces, is within the scope of the same pull request.